packages/backend/src/api/controllers/OldForcedTradeOfferController.test.ts

```diff
@@ -9,10 +9,10 @@ import { PositionRepository } from '../../peripherals/database/PositionRepositor
 import { UserRegistrationEventRepository } from '../../peripherals/database/UserRegistrationEventRepository'
 import { fakeAccepted, fakeOffer } from '../../test/fakes'
 import { mock } from '../../test/mock'
-import { ForcedTradeOfferController } from './ForcedTradeOfferController'
+import { OldForcedTradeOfferController } from './OldForcedTradeOfferController'
 import * as tradeMock from './utils/ForcedTradeOfferMockData'
 
-describe(ForcedTradeOfferController.name, () => {
+describe(OldForcedTradeOfferController.name, () => {
```

Comment:
Logic for forced trade offers hasn't really changed, so please reuse this unit tests to test our new implementation. We should reuse tests that we already have.